dask/bytes/__init__.py

```diff
@@ -3,24 +3,15 @@
 from fsspec.core import open as fs_open_file
 from fsspec.core import open_files as fs_open_files
 
+from ..utils import _deprecated
 from .core import read_bytes
 
 
+@_deprecated(use_instead="fsspec.core.open_file")
```

Comment:
```suggestion
@_deprecated(use_instead="fsspec.core.open")
```

Reply:
Whoops, I missed the `as open_file` part in the previous warning message. Thanks for catching this!